### DISCUSSION TITLE
Sloppiness in Free Form Tool

### DISCUSSION THREAD

**Discussion by khammerer** [excalidraw/excalidraw#3973] [Q&A]
Is there a way to adjust the sloppiness of the stroke line in the freeform tool (Draw - 7)? In particular, it would be useful to be able to set the stroke width to a single line, as can be done in the other tools.

**Comment by dwelle** [comment]
It is already a single line. Do you mean disabling the emulated pressure (variable width)?

**Reply by khammerer** [reply]
In the meantime, I have solved the problem. The problem concerned the use of Excalibur in Logseq. Interestingly, there is a menu item "Sloppiness" for the freeform there, which is not present in the Wep App of Excalidraw. It took me a while to discover this menu...

Thank you for the prompt response!
